openquake/commands/submit.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+#
+# Copyright (C) 2019 GEM Foundation
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import json
+import requests
+from openquake.baselib import sap, config
+from openquake.hazardlib import valid
+
+
+@sap.script
+def submit(job_ini, hc=None):
+    """Post a local file to the WebUI"""
+    job_ini = os.path.abspath(job_ini)
+    dic = dict(job_ini=job_ini, hazard_job_id=hc)
+    # NB: there is no WebUI port in openquake.cfg for the moment
```

Comment:
We need to add it it to the `openquake.cfg` but in a 'client' sections: in real cases the 'client' port (i.e. 443) is different from the daemon port (i.e. 8800) because a reverse proxy is used. We also need to specify if we need to make the connection via HTTPS or plain HTTP.

Reply:
Can you add that, Daniele?

Reply:
An alternative would be to pass on the command line the server URL.
It looks better to me, what do you think?